apps/changing_start_command.go

```diff
@@ -35,6 +35,9 @@ var _ = AppsDescribe("Changing an app's start command", func() {
 	Context("by using the command flag", func() {
 		var expectedNullResponse string
 
+		// TODO: No binary buildpack w/o Procfile support
```

Comment:
Is the idea here that we don't have support for the -c flag with a procfile regardless? And the cli previously only allowed us to use the -c flag with the binary buildpack?

Reply:
I think it is more that we simply don't have support for custom start commands at all. You have to use a Procfile.